frontend/packages/client/src/pages/Community.js

```diff
@@ -156,6 +158,17 @@ export default function CommunityPage() {
 
   const { data: strategies } = useVotingStrategies();
 
+  const {
+    pagination: { totalRecords },
+  } = useCommunityMembers({ communityId });
+  const [totalMembers, setTotalMembers] = useState();
+  useEffect(() => {
+    setTotalMembers(totalRecords);
+  }, [totalRecords]);
+
+  // for now allowList just returns admin addresses
+  const { data: adminAddrs } = useAllowlist();
```

Comment:
you could reuse the return value from there: https://github.com/DapperCollectives/CAST/pull/17/files#diff-23488fb68fdc6db2dd3f17eab17a1a0858d02c54d55aa5852e7479b5bb385fc3R156
 `const { data: admins } = useCommunityUsers({ communityId, type: "admin" });`